crates/lox_core/src/bodies/barycenters.rs

```diff
@@ -1,11 +1,3 @@
-/*
- * Copyright (c) 2023. Helge Eichhorn and the LOX contributors
- *
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
-
```

Comment:
The generator needs to be updated to generate the copyright statements. I'll add this in a separate PR.

Reply:
https://github.com/lox-space/lox/pull/9/files